src/utils/attribution-html.ts

```diff
@@ -1,70 +1,191 @@
 import type { Media } from '~/models/media'
```

Comment:
Very nice cleanup of the attribution creation function!
Do you think we should also open an issue for internationalizing it? It would be out of scope for this PR.